azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py

```diff
@@ -211,6 +212,7 @@ def __init__(
         self.ingest_by_tags = ingest_by_tags
         self.drop_by_tags = drop_by_tags
         self.flush_immediately = flush_immediately
+        self.ignore_first_record = ignore_first_record
```

Comment:
The name of the ingestion properties json rep for this value is 'ignoreFirstRecord'
What makes this be written correctly when sending the request.

Can we add an E2E test to make sure it works as expected?
there are already E2Es ingesting CSVs so it should be simple

Reply:
Not sure i understood correctly - this attribute will receive data from outside, not directly from the JSON - so the naming here doesnt really matter (as long as the JSON is being read correctly somewhere else). did you mean something different?

Reply:
Hey yochai, we are using the library inflection to convert the case from camelcase to snakecase - https://github.com/Azure/azure-kusto-python/blob/ronmoneta/ignorefirestrecord/quick_start/sample_app.py#L39

We don't need e2e tests here I think, or if we did - this change doesn't warrants them.

I'm resolving